modules/ROOT/pages/configuration/configuration-settings.adoc

```diff
@@ -2375,6 +2375,52 @@ m|+++false+++
 |===
 
 
+[role=label--new-5.21]
+[[config_db.index.fulltext.eventually_consistent_apply_parallelism]]
+=== `db.index.fulltext.eventually_consistent_apply_parallelism`
+
+.db.index.fulltext.eventually_consistent_apply_parallelism
+[frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
+|===
+|Description
+a|The number of threads that are applying the queued index updates for eventually consistent full-text indexes.
```

Comment:
This also needs to be changed and merged in the codebase before the cut-off on Friday.